web/js/animation/anim.js

```diff
@@ -1,6 +1,6 @@
 import lodashEach from 'lodash/each';
 
-export function parse (state, errors) {
+export function parse(state, errors) {
```

Comment:
Good change, I prefer the no space syntax. I think the space syntax was a left over rule from conforming to semi-standard.